src/CHANGES.txt

```diff
@@ -6,6 +6,11 @@
 
 RELEASE  VERSION/DATE TO BE FILLED IN LATER
 
+  From Adam Gross:
+
+    - Fixed duplicate "Preserved directory X" messages when running unit tests
+      with PRESERVE=1 in the environment.
```

Comment:
Please change "unit tests" to "SCons unit tests" just to avoid any confusion that it might impact other non-scons tests which might be run by SCons in a users build.